assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -180,7 +180,10 @@ function FrameElement({ element }) {
         ref={elementRef}
         data-element-id={id}
         {...box}
-        tabIndex="0"
+        // TODO: Investigate
```

Comment:
Hm, I think this is for being able to tab into an element to select it, however, it doesn't really seem to be working as expected currently at all when testing. So this might need some rethinking or a separate accessibility issue (unless it's just me).